src/FluentCMS.Domain/Services/SiteService.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Ardalis.GuardClauses;
+using FluentCMS.Entities.Sites;
+using FluentCMS.Repository;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FluentCMS.Services;
+public class SiteService(ISiteRepository siteRepository)
+{
+    private readonly ISiteRepository _siteRepository = siteRepository;
```

Comment:
When you are using primary constructors, you dont need to use this field in your class.